site/en/docs/extensions/mv3/promises/index.md

```diff
@@ -158,6 +157,8 @@ chrome.tabs.create({...}, (result) => {
 });
 ```
 
+#### Error handling with promises
+
 Promises are designed to deliver asynchronous results, both success and failure.  A failure in a
 promise (a promise rejection) is handled differently.  It might look like this:
 
```

Comment:
We may want to mention after this example something like "Alternatively, the error handler can be passed as a second argument to the .then() method".

Reply:
I'd rather link to e.g. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise than get into the business of fully documenting `Promise`.

Reply:
Done. PTAL.